src/ontology/mondo-edit.obo

```diff
@@ -512847,6 +512847,9 @@ synonym: "DEE64" EXACT ABBREVIATION [DOID:0070375, OMIM:618004]
 synonym: "developmental and epileptic encephalopathy 64" EXACT [DOID:0070375, OMIM:618004]
 synonym: "EIEE64" EXACT ABBREVIATION []
 synonym: "epileptic encephalopathy, early infantile, 64" EXACT [OMIM:618004]
+synonym: "RHOBTB2 syndrome" EXACT []
+synonym: "RHOBTB2-associated neurodevelopmental disorders" EXACT [PMID:37165955]
+synonym: "RHOBTB2-related disorders" EXACT [PMID:37090824]
```

Comment:
This one should be a broad synonym. 
For example, you could have a "new" disorder found that is RHOBTB2-related, but not neurodevelopmental. Even though we might not have someone "now", it is possible one would be found later.